### PR TITLE
Added covid tracker 

### PR DESCRIPTION
**Pull request by madhavtripathi05**

### CODE REVIEW

**Comment by madhavtripathi05** [comment]
- [X] I have read the Code Of Conduct.
- [X] I have followed all the steps of submission properly.

**Comment by jayskhatri** [comment]
Thanks @madhavtripathi05 , for opening the pull request! 👏

I'll be reviewing your pull request thoroughly

Star ⭐ this project and [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions and also you can join the [discord channel of club gamma](https://discord.gg/kjnp6wU)

File: madhavtripathi05/main.dart [hunk 1]
```diff
@@ -0,0 +1,9 @@
+void main(List<String> args) {
+  String profileUrl = 'https://github.com/madhavtripathi05';
+  var projectLinks = [
```
Comment (jayskhatri) [inline madhavtripathi05/main.dart]: Hey @madhavtripathi05 please add your code in here itself.

**Review by jayskhatri — CHANGES_REQUESTED** [review]
PR will be merged after you update the changes

File: madhavtripathi05/main.dart (hunk 1 above)
Comment (madhavtripathi05) [reply madhavtripathi05/main.dart]: okay will do

**Comment by madhavtripathi05** [comment]
done

**Review by jayskhatri — APPROVED** [review]
Good to go👍

Star ⭐ this project and [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions and also you can join the [discord channel of club gamma](https://discord.gg/kjnp6wU)

**Comment by yashpaneliya** [comment]
Hello contributor👋,

As we all know that Hacktoberfest-2020 is ended. Hope you enjoyed it a lot and made some awesome contributions. 
So now it's time for schwags🎁🤩

**Send your email (linked with your GitHub account) in the reply of this comment. Mention any one of the maintainers (@yashpaneliya @rudrabarad @harshptl14 @chirayu-joshi @dhruvpatel1706 ) in the reply.**

**We'll send a form on the email you'll provide. Please reply ASAP.**

Join [Club Gamma Discord](https://discord.com/invite/kjnp6wU) for future updates.

**Comment by madhavtripathi05** [comment]
> Hello contributor👋,
> 
> As we all know that Hacktoberfest-2020 is ended. Hope you enjoyed it a lot and made some awesome contributions.
> So now it's time for schwags🎁🤩
> 
> **Send your email (linked with your GitHub account) in the reply of this comment. Mention any one of the maintainers (@yashpaneliya @rudrabarad @harshptl14 @chirayu-joshi @dhruvpatel1706 ) in the reply.**
> 
> **We'll send a form on the email you'll provide. Please reply ASAP.**
> 
> Join [Club Gamma Discord](https://discord.com/invite/kjnp6wU) for future updates.

madhav.37.tripathi@gmail.com @harshptl14

**Pull request merged**